docs/data/data-grid/row-height/row-height.md

```diff
@@ -78,6 +77,15 @@ Add padding to the cells to increase the space between the content and the cell
 
 :::
 
+### Column virtualization
+
+By default, the virtualization of the columns is disabled to force all columns to be rendered at the same time and calculate the row height correctly.
```

Comment:
```suggestion
By default, the virtualization of the columns is disabled to force all columns to be rendered at the same time and calculate the row height correctly.
However, this can lead to poor performance when rendering a lot of columns.
```